DQM/Integration/python/clients/hlt_dqm_sourceclient-live_cfg.py

```diff
@@ -104,6 +104,9 @@
 ### for Proton-Lead collisions only (2016 Proton-Lead Era)
 #process.load("DQM.HLTEvF.HLTObjectMonitorProtonLead_cff")
 
+### for pp collisions 5 TeV pp reference run (2017 ppRef Era)
+#process.load("DQM.HLTEvF.HLTObjectMonitorPPReference_cff")
```

Comment:
uhmmm, I think you need to have this line un-commented !!!

Reply:
Will un-comment in the back port.